### PR TITLE
Update app_tr.arb

### PR DESCRIPTION
**Pull request by mikropsoft**

### CODE REVIEW

File: lib/l10n/l10n.dart [hunk 1]
```diff
@@ -7,10 +7,12 @@
 /// TexturedPolak@github => Polish
 /// yuri-val@github => Ukrainian
 /// energywave@github, ncvescera@github, OpenCode@github => Italian
-/// mdksec@github => Turkish
+/// mikropsoft@github => Turkish
```
Comment (KRTirtho) [inline lib/l10n/l10n.dart:10]:
```suggestion
/// mdksec@github, mikropsoft@github => Turkish
```

You can't remove the previous contributor but you can add your name next to his/her name

**Review by KRTirtho — CHANGES_REQUESTED** [review]

File: lib/l10n/l10n.dart (hunk 1 above)
Comment (mikropsoft) [reply lib/l10n/l10n.dart]: that's my first username, that's why I changed it @KRTirtho

Comment (mikropsoft) [reply lib/l10n/l10n.dart]: > that's my first username, that's why I changed it @KRTirtho

I am the one who sent the file through the community :)

Comment (KRTirtho) [reply lib/l10n/l10n.dart]: Just because you're reborn doesn't mean your old existence has to be erased /s

Kidding, I'll change it back 😄

**Pull request merged**